core/getpoint/run_summarization.py

```diff
@@ -138,7 +139,7 @@ def convert_to_coverage_model():
   # load all non-coverage weights from checkpoint
   saver = tf.train.Saver([v for v in tf.global_variables() if "coverage" not in v.name and "Adagrad" not in v.name])
```

Comment:
Some lines look too long that need to be resolved, but once checked, LGTM.

Reply:
We do not reorganize code styles in the underlying code, and we have explicitly ignored them in our CI.